Src/FluentAssertions/Primitives/DateTimeAssertions.cs

```diff
@@ -168,12 +168,18 @@ public AndConstraint<TAssertions> Be(DateTime? expected, string because = "", pa
         long distanceToMaxInTicks = (DateTime.MaxValue - nearbyTime).Ticks;
         DateTime maximumValue = nearbyTime.AddTicks(Math.Min(precision.Ticks, distanceToMaxInTicks));
 
+        var difference = (Subject - nearbyTime)?.Duration();
```

Comment:
`...Duration()`: 
I was not sure what fits better: the "real" difference (meaning this also could be negative) or the absolute values..

Reply:
In this case, the value can be off in both directions, so an absolute value is enough.

Reply:
👍🏻